gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/setWriter.ts

```diff
@@ -57,22 +57,51 @@ export default class SetWriter extends SolanaCommand {
 
     console.log(`Setting store writer on Store (${storeState.toString()}) and Feed (${feedState.toString()})`)
```

Comment:
Move this to execution as well

Reply:
Moved, but had to recalculate some of the variables. We might need shared state/variables to be shared across execute/makeRawTransaction or some other way to tackle it.

Might be related to https://github.com/smartcontractkit/chainlink-solana/issues/127, to add more shared state.